pulsar-client-cpp/lib/stats/ConsumerStatsImpl.h

```diff
@@ -30,30 +30,37 @@ class ConsumerStatsImpl : public ConsumerStatsBase {
    private:
     std::string consumerStr_;
 
-    unsigned long numBytesRecieved_ = 0;
-    std::map<Result, unsigned long> receivedMsgMap_;
-    std::map<std::pair<Result, proto::CommandAck_AckType>, unsigned long> ackedMsgMap_;
+    ExecutorServicePtr executor_;
+    DeadlineTimerPtr timer_;
+    unsigned int statsIntervalInSeconds_;
+
+    struct AsyncCallbackLock;
+    std::shared_ptr<AsyncCallbackLock> callbackLock_;
 
+    std::mutex rcvMutex_;
+    unsigned long numBytesRecieved_ = 0;
     unsigned long totalNumBytesRecieved_ = 0;
```

Comment:
I think we can correct the original typo as well? `numBytesRecieved_` -> `numBytesReceived_`.

Reply:
Renamed to numBytesReceived_